.github/workflows/pytest.yaml

```diff
@@ -81,9 +81,19 @@ jobs:
       run: |
         pip install .[test] -c requirements.txt
       if: ${{ !matrix.min-version }}
+    - name: Install node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '20.x'
+    - name: install pyright
+      run: |
+        npm install
+        echo "$PWD/node_modules/.bin" >> $GITHUB_PATH
```

Comment:
Document that npm does not support installing packages globally from a package.json file which is why we do this path modification

Reply:
This also needs to be updated to support windows